javascript/chronometer.js

```diff
@@ -1,26 +1,59 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0
+    this.intervalId = ''
   }
   startClick(callback) {
-    // ... your code goes here
+    this.intervalId = setInterval(() => {
+      if (callback) {
+      callback()}
+      if (this.currentTime > 100) {
+        this.currentTime = 0
+      }
+      this.currentTime +=0.01
+    }, 10)
   }
   getMinutes() {
-    // ... your code goes here
+    let time = this.currentTime
+    let minutes = 0
+    if (time >= 60) {
+      minutes = time / 60
+    }
```

Comment:
this passes the test, but later on the feedback you will see that the idea is that this function returns the minutes even if the seconds are still lower than 60, so returning 0 minutes. This else condition will be needed.